src/main/java/com/rocketden/main/controller/v1/RoomController.java

```diff
@@ -42,6 +43,12 @@ public RoomController(RoomService service) {
         return new ResponseEntity<>(service.createRoom(request), HttpStatus.CREATED);
     }
 
+    @DeleteMapping("/rooms/{roomId}/user/{userId}")
+    public ResponseEntity<RoomDto> removeUser(@PathVariable String roomId,
+                                              @PathVariable String userId) {
+        return new ResponseEntity<>(service.removeUser(roomId, userId), HttpStatus.OK);
+    }
```

Comment:
I think you should add another RequestBody variable that stores the initiator information - that way only the host is allowed to remove users from a room

Reply:
Sure, I think it's a great idea to put the host ID in! I've tried having a body on a delete request before and it's not quite the same as on other types of requests, but I'll see if I can make it work.